backend/utils/db_utils.py

```
@@ -0,0 +1,376 @@
+    for pref in preferences:
+        cursor.execute(
+            """
+            UPDATE UserPreferences
+            SET PreferredView = ?, Enabled = ?
+            WHERE SourceID = ? AND ViewName = ?
+            """,
+            (pref.preferredView, int(pref.enabled), source_id, pref.viewName),
+        )
+
+
+def fetch_alert_summary(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)